### PR TITLE
Created a solution pack -  SOC Overview)

### PR DESCRIPTION
**Pull request by cs-yash-patodkar**
Problem Statement:
Currently, we need to manually install and configure SOC management widget.Expected: We can create new Solution Pack with below requirements
1. Install SOC Management widget
2. Configure SOC visualisation Dashboard with default values :
Where - 'Created On' Last 7 Days

Description :
The Solution contains a Dashboard which has widget(Soc Management)
The Solution pack will install the soc management Widget if not already installed and create Dashboard With the Widget

Test Case:
1. Installed the SP
2. The Widget gets installed 
3. A dashboard is created with the widget

PMDB : https://pmdb.fortinet.com/ProjectManagement/viewDocument.php?id=21297

### CODE REVIEW

File: dashboards/SOC Management Dashboard.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "@context": "\/api\/3\/contexts\/Dashboard",
+    "@type": "Dashboard",
+    "uuid": "57ffafc7-bd34-4b0f-b126-2c91fc2b0860",
+    "displayName": "SOC Management Dashboard",
+    "type": "rows",
+    "templateType": "dashboard",
+    "roles": [],
```
Comment (cs-sujeet-ahirrao) [inline dashboards/SOC Management Dashboard.json]: Add Roles to the Dashboard

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- In a single image it displays the overall picture of the investigations carried out by a SOC analyst, such as, how many alerts were created, the type of alerts, how many alerts of the total were closed, categorized as False Positive, True Positive, etc, in the time frame you have specified. 
+- It also displays information about the top three playbooks, data sources, alert types, the ROI, playbooks run, etc., and also some information about the incidents including the top three incident types, their associated assets, and analyzed artifacts.
+
+# Next Steps
+
+Use absolute links
+
+| [Installation](https://github.com/fortinet-fortisoar/solution-pack-soc-overview/blob/release/1.0.0/docs/setup.md#installation) | [Configuration](https://github.com/fortinet-fortisoar/solution-pack-soc-overview/blob/release/1.0.0/docs/setup.md#configuration) | [Usage](https://github.com/fortinet-fortisoar/solution-pack-soc-overview/blob/release/1.0.0/docs/usage.md) | [Contents](https://github.com/fortinet-fortisoar/solution-pack-soc-overview/blob/release/1.0.0/docs/contents.md) |
```
Comment (cs-sujeet-ahirrao) [inline README.md]: These links are not working

Comment (cs-yash-patodkar) [reply README.md]: Resolved the error, updated the links

File: dashboards/SOC Management Dashboard.json (hunk 1 above)
Comment (cs-yash-patodkar) [reply dashboards/SOC Management Dashboard.json]: Added roles, Full app permission, SOC analyst, SOC manager !

**Review by cs-sujeet-ahirrao — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Release Information
+
+* **Version**: 1.0.0
+* **Certified**: No
+* **Publisher**: Fortinet
+* **Compatible Version**: FortiSOAR v7.2.2
```
Comment (cs-suyog-jinturkar) [inline README.md:6]: Why 7.2.2 only?

Comment (cs-suyog-jinturkar) [reply README.md:6]: info.json has value --> "fsrMinCompatibility": "7.2.0",

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added one small comment

File: README.md (hunk 3 above)
Comment (cs-yash-patodkar) [reply README.md:6]: The SOC management widget has a minimum compatibility of 7.2.2

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**